sdks/python/apache_beam/utils/windowed_value.py

```diff
@@ -178,34 +182,22 @@ def __repr__(self):
         self.windows,
         self.pane_info)
 
+  def _key(self):
+    return self.value, self.timestamp_micros, self.windows, self.pane_info
+
+  def __eq__(self, other):
+    return (type(self) == type(other)
+            and self.timestamp_micros == other.timestamp_micros
+            and self.value == self.value
```

Comment:
s/self/other in right hand side of the equality.

Reply:
Sorry, took this straight from the benchmark and didn't notice it.

Reply:
No worries, thank you.